spec/functions/validate_slength_spec.rb

```diff
@@ -1,67 +1,61 @@
-#! /usr/bin/env ruby -S rspec
-
 require 'spec_helper'
 
-describe "the validate_slength function" do
-  let(:scope) { PuppetlabsSpec::PuppetInternals.scope }
-
-  it "should exist" do
-    expect(Puppet::Parser::Functions.function("validate_slength")).to eq("function_validate_slength")
+describe 'validate_slength' do
```

Comment:
gosh, i'm so looking forward when these kind of functions go the way of the dinosaur\*.
*****
\* turn into birds, and fly away. away from our code.